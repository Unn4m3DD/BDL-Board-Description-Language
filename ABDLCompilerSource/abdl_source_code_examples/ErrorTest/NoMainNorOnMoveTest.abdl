int other_function():
  return 0;
end other_function;