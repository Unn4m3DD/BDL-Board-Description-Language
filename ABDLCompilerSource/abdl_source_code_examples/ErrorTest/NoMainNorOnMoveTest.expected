Neither main or on_move declared